generators/entity-server/templates/ui/widgets/tableWidget/src/api/entityApi.js.ejs

```diff
@@ -0,0 +1,17 @@
+<%_
+let entityNameLower = entityClass.charAt(0).toLowerCase() +  entityClass.slice(1)
```

Comment:
Could you change this to entityInstance?

Reply:
Fixed